src/current/_data/redirects.yml

```diff
@@ -227,40 +170,80 @@
   sources: ['view-version-details.html']
   versions: ['v19.2', 'v20.1', 'v20.2', 'v21.1']
 
+- destination: cockroachdb-feature-availability.html
+  sources: 
+    - experimental-features.html
+    - features-in-preview.html
+  versions: ['v22.1', 'v22.2', 'v23.1']
+
+- destination: common-errors.html
+  sources: ['general-troubleshooting.html']
+  versions: [':version']
+
+- destination: connect-to-a-serverless-cluster.html
+  sources: ['connect-to-a-free-cluster.html']
+  versions: ['cockroachcloud']
+
+- destination: cost-based-optimizer.html
+  sources: ['sql-optimizer.html']
+  versions: [':version']
+
+- destination: create-a-serverless-cluster.html
+  sources: ['create-a-free-cluster.html']
+  versions: ['cockroachcloud']
+
 - destination: demo-low-latency-multi-region-deployment.html
   sources: ['demo-geo-partitioning.html']
   versions: ['v19.2', 'v20.1', 'v20.2', 'v21.1']
 
 - destination: demo-replication-and-rebalancing.html
-  sources: ['demo-automatic-rebalancing.html', 'demo-data-replication.html']
+  sources: 
+  - demo-automatic-rebalancing.html
+  - demo-data-replication.html
   versions: ['v19.2', 'v20.1', 'v20.2', 'v21.1']
 
 - destination: example-apps.html
   sources: ['build-an-app-with-cockroachdb.html']
   versions: ['v21.2']
 
+- destination: fips.html
+  sources: ['fips-compliance.html']
+  versions: ['v23.1']
```

Comment:
This is weird because it will also apply to all future versions. I wonder if the version should just be 'stable' here.

Reply:
If for some reason we ever link to `v23.1/fips-compliance.html` or `dev/fips-compliance.html`, I believe this kind of redirect is necessary.